pages/common/speedtest-cli.md

```diff
@@ -1,32 +1,36 @@
 # speedtest-cli
 
-> Command line interface for testing internet bandwidth using https://speedtest.net.
+> Unofficial command line interface for testing internet bandwidth using https://speedtest.net.
 > More information: <https://github.com/sivel/speedtest-cli>.
```

Comment:
Perhaps this will read better?
```suggestion
> Unofficial command line interface for testing internet bandwidth using https://speedtest.net.
> See also `speedtest`, the official CLI.
> More information: <https://github.com/sivel/speedtest-cli>.
```